src/sentry/migrations/0110_sentry_app_creator_backill.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-06 17:57
+from __future__ import unicode_literals
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+from django.db import migrations
+
+def backfill_one(audit_log_entry, SentryApp):
+    name = audit_log_entry.data.get("sentry_app")
+    if name:
+        # find the sentry app for that org, name, that's not deleted
+        sentry_app = SentryApp.objects.filter(
+            name=name,
+            owner=audit_log_entry.organization,
+            date_deleted__isnull=True,
+            creator_user__isnull=True,
+        ).first()
+        # if there is a match and the user exists,
+        # update with the creator
+        user = audit_log_entry.actor
+        if sentry_app and user:
+            sentry_app.creator_user = user
+            sentry_app.creator_label = user.email or user.username
+            sentry_app.save()
+
+
+def backfill_sentry_app_creator(apps, schema_editor):
+    """
+    Backills the creator fields of SentryApp from
+    the audit log table
+    """
+    SentryApp = apps.get_model("sentry", "SentryApp")
+    AuditLogEntry = apps.get_model("sentry", "AuditLogEntry")
+
+    queryset = AuditLogEntry.objects.filter(event=113) # sentry app add
```

Comment:
Remove this filter. Unless you know for a fact you have an index on this. Otherwise, it'll take forever to run.

Reply:
There's no index from what I can tell, so having the filter here is pretty detrimental. It has to be removed and do the filter in python. We have to iterate all rows in the table.

Reply:
@mattrobenolt Got it. What if I did something like this instead:
```
AuditLogEntry.objects.filter(event=113, organization_id=org_id)
```
Since we have an index on `organization_id`. But I would have to do this for every Sentry App which is like 5k rows. Thoughts?

Reply:
The `event=113` bit isn't going to help anything fwiw, but the `organization_id` bit would be better. So yeah, I think tackling it from the otherside and just getting AuditLogEntry records from the relevant org ids would be more efficient.